models/activities/notification.go

```diff
@@ -717,6 +828,31 @@ func (nl NotificationList) LoadComments(ctx context.Context) ([]int, error) {
 	return failures, nil
 }
 
+func (nl NotificationList) LoadReleases(ctx context.Context) ([]int, error) {
+	if len(nl) == 0 {
+		return []int{}, nil
+	}
+
+	failures := []int{}
+	for i, notification := range nl {
+		if notification.Source != NotificationSourceRelease {
+			continue
+		}
+
+		release, err := repo_model.GetReleaseByID(ctx, notification.ReleaseID)
```

Comment:
It's low efficiency. We need to `id in ()` to fix the possible performance problem.

Reply:
`GetReleaseByID` does not more than doing a Database query with the ID. It does not load other things if it's that what you mean.

Reply:
You can find many examples to a batch loading. Use a for loop to get all release id and then get all releases use `id in ()` and then assign release to notification.